### PR TITLE
wrote a new format currency method which will localize any number passed in

### PR DESCRIPTION
**Pull request by ckhatri**
Currently My Accounts: Currency localization is different than the values shown in the Price and market cap section, so this method will take in any value and localize it so we have consistency.

### CODE REVIEW

File: client/app/src/accounts/account.controller.js [hunk 1]
```diff
@@ -586,12 +586,30 @@
...
 
       var currencyBalance = Number(balance * price).toLocaleString(languageCode, options);
-      
+
       if (currencyName == "btc") currencyBalance = currencyBalance.replace("BTC", "Ƀ");
 
       return currencyBalance;
     }
 
+    self.formatCurrencyVal = function(val) {
```
Comment (luciorubeens) [inline client/app/src/accounts/account.controller.js]: `formatCurrencyBalance` and `formatCurrencyVal` sounds very similar, can you merge both into a single function?

File: client/app/index.html [hunk 2]
```diff
@@ -177,7 +177,7 @@
                   <b><translate>Price</translate></b>
                 </div>
 
-                <span class="md-secondary">{{ul.currency.symbol}}{{ul.connectedPeer.market.price[ul.currency.name]}}</span>
+                <span class="md-secondary">{{ul.formatCurrencyVal(ul.connectedPeer.market.price[ul.currency.name])}}</span>
```
Comment (spresnal) [inline client/app/index.html]: These look like good use cases for a custom angular filter

Comment (luciorubeens) [reply client/app/index.html]: yeah! I agree @spresnal

**Comment by ckhatri** [comment]
create a myAccountsCurrencyBalance to get the value of the account in whichever currency, and then pass that into formatCurrencyVal to format it.

**Comment by ckhatri** [comment]
Created the custom filter and implemented it!

**Comment by ckhatri** [comment]
there was a merge conflict, so I went ahead and fixed it

**Comment by ckhatri** [comment]
should be good to go now

**Comment by luciorubeens** [comment]
nice +5 👍🏻

**Pull request merged**